src/javascript/pages/cashier/payment_methods.js

```diff
@@ -4,24 +4,145 @@ const Client         = require('../../common/client');
 const CashierPaymentMethods = (function() {
     'use strict';
 
-    const hidden_class = 'invisible';
+    const VIEWPORT_TABS = 6;
+    let $container;
+    let $previousButton;
+    let $nextButton;
+    let isVertical;
+    let currentFirstTab;
 
     const load = () => {
         ChampionSocket.wait('authorize').then(() => {
-            const container = $('.fx-payment-methods');
+            $container = $('.scrollable-tabs');
+            $previousButton = $('.previous-button');
+            $nextButton = $('.next-button');
+            isVertical = $(window).innerWidth() < 767;
+            currentFirstTab = 1;
+
+            const container = $('#payment_methods');
```

Comment:
Having two variables with the same name `$container` and `container` is ambiguous, also the second one is a jQuery object as well, so better to start with `$` for consistency.